src/algorithms.jl

```diff
@@ -94,7 +94,7 @@ end
 
 function ImplicitEulerExtrapolation(;chunk_size=Val{0}(),autodiff=true, standardtag = Val{true}(), concrete_jac = nothing,
     diff_type=Val{:forward},linsolve=nothing,precs = DEFAULT_PRECS,
-    max_order=12,min_order=3,init_order=5,threading=true,sequence = :bulirsch)
+    max_order=12,min_order=3,init_order=5,threading=false,sequence = :harmonic)
```

Comment:
why default to false while the others default to true?

Reply:
Extrapolation methods have default threading to false. https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/algorithms.jl#L280

Reply:
https://github.com/SciML/OrdinaryDiffEq.jl/blob/master/src/algorithms.jl#L79 missed one then.